AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/resources/ResourceTypesRest.java

```diff
@@ -71,4 +81,24 @@ public List<ResourceTypeDTO> getRootResourceTypes() {
                 .map(ResourceTypeDTO::new)
                 .collect(Collectors.toList());
     }
+
+    @Path("/resourceTypes")
+    @POST
+    @ApiOperation(value = "Add a new resource type")
+    @Consumes("application/json")
+    public Response addNewResourceType(ResourceTypeRequestDTO request)
+            throws ResourceTypeNotFoundException, ElementAlreadyExistsException, ValidationException {
```

Comment:
Exception for ResourceTypeNotFoundException should be wrapped in NotFoundException.

Reply:
Add NotFoundException to method return-types